htdocs/sass/rcloud-discover.scss

```diff
@@ -99,6 +99,10 @@ progress::-webkit-progress-value {
 .grid-item h1{
     font-size: 17px;
     margin: 0;
+    /* firefox */
+    white-space: pre-wrap;
+    word-break: break-all;
+    /* chrome */
     word-break: break-word;
```

Comment:
Sorry, I was unclear! I meant that I think there's no need to special case Chrome with something non-standard when the standard `word-break: break-all` seems to work on both browsers.